### ISSUE TITLE
How to do ensemble with ivadomed?

### ISSUE THREAD

**Issue by jcohenadad** [ivadomed/ivadomed#1285]
### Motivation for the feature

To make model in production more performant, we might want to do ensembling. Eg: Bagging, where multiple models are trained, each with bootstrapped train/validation dataset, and then we fuse all predictions at test time (eg using averaging or STAPLE). 

Is there such feature available in ivadomed? How were past ensembling projects performed with ivadomed. Input from @charleygros and @andreanne-lemay would be greatly appreciated 🙏

**Comment by naga-karthik** [comment]
If it helps, for the Loss ensembles section in the [label fusion paper](https://www.melba-journal.org/pdf/2022:031.pdf), we trained multiple models with different seeds and then averaged them after all the training was done. There was no special "feature" of ivadomed that was used.

**Comment by jcohenadad** [comment]
Thank you for chipping in Naga!

> we trained multiple models with different seeds and then averaged them after all the training was done.

I thought what was averaged was the predictions, not the models?

**Comment by naga-karthik** [comment]
Yes, you're right, the predictions from each model were obtained and then averaged. 

Hmm, I may have misunderstood your question then. I don't know how we can average "models" because what we are interested in obtaining from each model are its predictions, and we usually average the predictions from different models.

**Comment by jcohenadad** [comment]
You understood it right-- my question was very pragmatic: is there a feature to do ensemble/bagging with ivadomed, and from what you're saying, the answer is no (ie: the averaging was done 'outside' of ivadomed, using code specific to the label fusion project)-- am i right?

**Comment by naga-karthik** [comment]
Yes, you're right. Ensembling can't be done with ivadomed right now. We would have to do it on our own by taking ivadomed's predictions.

**Comment by charleygros** [comment]
> Yes, you're right. Ensembling can't be done with ivadomed right now. We would have to do it on our own by taking ivadomed's predictions.

I second this, yes
